xgcm/test/test_transform.py

```diff
@@ -47,3 +47,26 @@ def test_interp_1d_conservative():
     dz_theta = interp_1d_conservative(dz_2d, theta_2d, theta_bins)
 
     np.testing.assert_allclose(dz_theta.sum(axis=-1), dz.sum(axis=-1))
+
+
+"""
+Dummy tests for grid object methods.
+"""
+
+
+@pytest.fixture(scope="module", params=["a", "b", "c"])
+def dataset(request):
+    datasets = {
+        "a": (5, {"opt1": "a"}, {"opt1": "a"}, 10),
+        "b": (4, {"opt1": "a"}, {"opt1": "a"}, 8),
+        "c": (4, {"opt1": "a"}, {"opt1": "a"}, 7),
+    }
+
+    input, grid_kwargs, transform_kwargs, expected = datasets[request.param]
+    return input, grid_kwargs, transform_kwargs, expected
+
+
+def test_dummy(dataset):
+    input, grid_kwargs, transform_kwargs, expected = dataset
+
+    assert input * 2 == expected
```

Comment:
@rabernat what do you think of this general setup? I will basically use the `dataset` fixture to build example datasets and the output I expect when passing the `grid_kwargs` to `Grid(input, **grid_kwargs)` and the `transform_kwargs` to the transform method `grid.transform(axis, transform_kwargs)`

Reply:
Yes, I like this structure a lot. 👍